system-tests/projects/protocol/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@packages/app-capture-protocol",
```

something like this maybe?
```suggestion
  "name": "protocol-sample-project",
```

---

Updated to `protocol-sample-project`: [73fef85](https://github.com/cypress-io/cypress/pull/27659/commits/73fef8513d89e7d92f15a74d51467f8ad42e086c)